# Fix for LSB compliance. Heartbeat relies on LSB behavior for correct high-availability
--- a/named     2009-08-20 16:50:54.000000000 -0400
+++ b/named     2009-08-20 17:09:52.000000000 -0400
@@ -133,6 +133,10 @@
                rm -f /var/lock/subsys/named
                rm -f /var/run/named.pid
            fi;
+        else
+            #At this point we know named is not running!
+            #Return 0 (fix for LSB compliance)
+            RETVAL=0
        fi;
         if [ -n "${ROOTDIR}" -a "x${ROOTDIR}" != "x/" ]; then
                 if egrep -q '^/proc[[:space:]]+'${ROOTDIR}'/proc' /proc/mounts; then
